config/locales/en.yml

```diff
@@ -30,5 +30,8 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
-  people: "People"
+    hello: 'Hello world'
+    people: 'People'
+    title-en: 'English'
+    title-de: 'German'
+    locale-test: 'This should be in englisch'
```

Comment:
```suggestion
    locale-test: 'This should be in english'
```

Reply:
The locale-test keys are actually unnecessary and have been removed